hack/grab-profiles.sh

```diff
@@ -18,17 +18,44 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+function grab_profiles_from_component {
+  local requested_profiles=$1
+  local mem_pprof_flags=$2
+  local binary=$3
+  local tunnel_port=$4
+  local path=$5
+  local output_prefix=$6
+  local timestamp=$7
+
+  echo "binary: $binary"
+
+  for profile in ${requested_profiles}; do
+    case ${profile} in
+      cpu)
+        go tool pprof "-pdf" "${binary}" "http://localhost:${tunnel_port}/${path}/profile" > "${output_prefix}-${profile}-profile-${timestamp}.pdf"
+        ;;
+      mem)
+        for flag in ${mem_pprof_flags}; do
+          go tool pprof "-${flag}" "-pdf" "${binary}" "http://localhost:${tunnel_port}/${path}/heap" > "${output_prefix}-${profile}-${flag}-profile-${timestamp}.pdf"
```

Comment:
Whatever those are :)
Anyway - can you please add a comment about it (also giving this example)?